tekton-catalog/cos-logger/README.md

```diff
@@ -0,0 +1,72 @@
+# cos-logger
+
+Log to cloud object storage for golang implemented as `io.Writer`.
+
+## Use it as a plugin/extension to [uber-go/zap](https://github.com/uber-go/zap) logger
+
+Configure logger and add a multi write syncer for `zap` as follows,
+
+```go
+package cos_logger
```

Comment:
thanks @ScrapCodes, Overall it looks good to me. There are some comments I have with the naming here.
1. Can we avoid using hyphens or underscores for naming? It looks like the golang community usually avoid using them. 
2. COS might be too specific for ibm cloud. Since this package also works on S3 and we might extend it with other storages in the future, can we rename the package to something like `objectstorelogger`? You probably want to rename the directory name as well.

Reply:
So a lot of the files and package names might need to change.